src/js/lib/client/package.js

```diff
@@ -21,11 +34,10 @@ let config = {
     },
 
     scripts: {
-        "start": `electron ${paths.dist.content} -- `,
-        "build": "webpack ",
-        "dist": "electron-builder --publish never",
-        "dist:crossplatform": "electron-builder --mac --win --linux --publish never"
-    }
+        start: `electron ${paths.dist.content} -- `,
+        build: 'webpack ',
+        dist: 'electron-builder --publish never' + ' --' + build.target,
```

Comment:
if your `get_build_config` fails, which it can, cause youre using if inside, then this could concatenate the string with "null" string, which is not what you want

Reply:
I've added an exception to `get_build_config` in case it can't load the build file.